test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -276,6 +276,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.index_management.enableIndexStats (any)',
         'xpack.index_management.editableIndexSettings (any)',
         'xpack.index_management.enableDataStreamsStorageColumn (any)',
+        'xpack.index_management.enableMappingsSourceField (any)',
```

Comment:
Can this actually be specified as boolean? 
```suggestion
        'xpack.index_management.enableMappingsSourceField (boolean)',
```

Reply:
Nevermind! I see now that it is an offering based schema.